scoring.js

```diff
@@ -11,33 +11,50 @@ const calculateScore = () => {
                 player.stats.rushing.fumbles * -3
             )
         case 'RB':
-            break
+            return (
+                player.stats.rushing.yards / 10 +
+                player.stats.rushing.touchdowns * 6 +
+                player.stats.rushing.fumbles * -3 +
```

Comment:
Don't forget DRY don't repeat yourself.  There is alot of repeating code that can be consolidated.